### PR TITLE
[Bluetooth] Add remain bluetooth functions

### PR DESCRIPTION
**Pull request by Wootak**
[Bluetooth] Add remain bluetooth functions

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [1fe0c5c389f0882cdb5f1049dfd2763ab00949d4] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1fe0c5c389f0882cdb5f1049dfd2763ab00949d4] - Code Rule Check Result:
os/include/tinyara/bluetooth/bt_hci.h:60: ERROR: [SPC_M_SEP] space required after that close brace '}'
os/include/tinyara/bluetooth/bt_hci.h:61: ERROR: [SPC_M_SEP] space required after that close brace '}'

**Comment by seinfra** [comment]
Target : [185bc79e5439c89fc987482565c8a8c1bbdc1b48] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [185bc79e5439c89fc987482565c8a8c1bbdc1b48] - Code Rule Check (C++) OK.

File: os/include/tinyara/bluetooth/bluetooth.h [hunk 1]
```diff
@@ -414,48 +432,48 @@ int bt_stop_advertising(void);
...
+ *  the specified callback.
  *
- * @param[in] filter_dups Enable duplicate filtering (or not).
- * @param[in] cb Callback to notify scan results.
+ *  @param param Scan parameters.
+ *  @param cb Callback to notify scan results.
  *
- * @return Zero on success or error code otherwise, positive in case
- * of protocol error or negative (POSIX) in case of stack internal error
+ *  @return Zero on success or error code otherwise, positive in case
```
Comment (sangam-swami) [inline os/include/tinyara/bluetooth/bluetooth.h]: can we use same spacing for all comments

File: os/include/tinyara/bluetooth/bt_core.h [hunk 2]
```diff
@@ -154,7 +162,7 @@ enum bt_security_e {
  *
  ****************************************************************************/
 
-static inline int bt_addr_to_str(FAR const bt_addr_t * addr, FAR char *str, size_t len)
```
Comment (sangam-swami) [inline os/include/tinyara/bluetooth/bt_core.h]: bt_addr_t *addr
Can we remove space

File: os/include/tinyara/bluetooth/bt_core.h [hunk 3]
```diff
@@ -177,7 +185,7 @@ static inline int bt_addr_to_str(FAR const bt_addr_t * addr, FAR char *str, size
  *
  ****************************************************************************/
 
-static inline int bt_addr_le_to_str(FAR const bt_addr_le_t * addr, char *str, size_t len)
```
Comment (sangam-swami) [inline os/include/tinyara/bluetooth/bt_core.h]: Same as above

**Comment by KimDongEon** [comment]
test

**Comment by Wootak** [comment]
I raised a new commit applying commit rule.
https://github.com/Samsung/TizenRT/pull/3523

**Pull request closed without merging**